README.md

```diff
@@ -70,53 +85,102 @@ In addition to publishing separate packages for each service client, each packag
-Configuration must be passed to each service client that is instantiated. 
+In the Developer Preview, there is no longer a global configuration managed by the SDK.
+Configuration must be passed to each service client that is instantiated.
 It is still possible to share the same configuration across multiple clients but that configuration will not be automatically merged with a global state.
+
 #### Middleware
+
 Version 2.x of the SDK allows modifying a request throughout multiple stages of a request’s lifecycle by attaching event listeners to a request.
 Some feedback we received frequently was that it can be difficult to debug what went wrong during a request’s lifecycle.
 We’ve switched to using a middleware stack to control the lifecycle of an operation call now.
 This gives us a few benefits. Each middleware in the stack calls the next middleware after making any changes to the request object.
 This also makes debugging issues in the stack much easier since you can see exactly which middleware have been called leading up to an error.
 Here’s an example of adding a custom header using middleware:
+
 ```javascript
 lambda.middlewareStack.add(
-  (next, context) => (args) => {
-    args.request.headers['Custom-Header'] = 'value';
+  (next, context) => args => {
+    args.request.headers["Custom-Header"] = "value";
     return next(args);
   },
   {
-    step: 'build'
+    step: "build"
   }
 );
 
 lambda.putObject(params);
 ```
-In the above example, we’re adding a middleware to our S3 client’s middleware stack. 
-The first argument is a function that accepts next, the next middleware in the stack to call, and context, an object that contains some information about the operation being called. 
+
+In the above example, we’re adding a middleware to our S3 client’s middleware stack.
+The first argument is a function that accepts next, the next middleware in the stack to call, and context, an object that contains some information about the operation being called.
 It returns a function that accepts args, an object that contains the parameters passed to the operation and the request, and returns the result from calling the next middleware with args.
+
+### Install from Source
+
+This repository is under development, not all service clients are available for install. If you want to try the clients that not yet published, you can follow this instruction to install these packages from source.
+
+1. Clone this repository to local by:
+
+    ```
+    git clone https://github.com/aws/aws-sdk-js-v3.git
+    ```
+
+1. Follow [these steps](https://github.com/aws/aws-sdk-js-v3/blob/master/CONTRIBUTING.md#setup-and-testing) to link and build the whole library, the process may takes several minutes.
+1. After the repository is successfully built, change directory to the client that you want to install, for example:
+
+    ```
+    cd clients/browser/client-lambda-browser
+    ```
```

Comment:
Should we add a step here that they need to run `yarn test` in the in client directory before packing?  It's a little hard to follow switching between the README and CONTRIBUTING when you have to switch back and forth between directories.